### PR TITLE
Command Summary - New UI

### PR DESCRIPTION
**Pull request by EyalDelarea**
- [x] All [tests](https://github.com/jfrog/jfrog-cli/CONTRIBUTING.md#tests) have passed. If this feature is not already covered by the tests, new tests have been added.
- [x] The pull request is targeting the `dev` branch.
- [x] The code has been validated to compile successfully by running `go vet ./...`.
- [x] The code has been formatted properly using `go fmt ./...`.

---

Loads possible scan results which were recorded in order to display them inside `buildInfoSummary`

image

### CODE REVIEW

File: general/summary/cli.go [hunk 1]
```diff
@@ -68,12 +67,15 @@ func GenerateSummaryMarkdown(c *cli.Context) error {
...
 	}
 
+	// Saves the final Markdown to the root directory of the command summaries
 	return saveMarkdownToFileSystem(finalMarkdown)
 }
 
+// The CLI generates summaries in sections, with each section as a separate Markdown file.
+// This function merges all sections into a single Markdown file and saves it in the root of the
+// command summary output directory.
 func combineMarkdownFiles() (string, error) {
```
Comment (eyalbe4) [inline general/summary/cli.go:77]: combineMarkdownFiles --> mergeMarkdownFiles

File: general/summary/cli.go [hunk 2]
```diff
@@ -37,14 +37,13 @@ func (ms MarkdownSection) String() string {
 	return string(ms)
 }
```
Comment (eyalbe4) [inline general/summary/cli.go:39]: Let's the below variable into a const - 
```
var markdownSections = []MarkdownSection{Security, BuildInfo, Upload}
```

File: utils/cliutils/commandsflags.go [hunk 3]
```diff
@@ -1829,7 +1829,7 @@ var commandFlags = map[string][]string{
 	},
 	Docker: {
 		buildName, buildNumber, module, Project,
-		serverId, skipLogin, threads, detailedSummary, watches, repoPath, licenses, xrOutput, fail, ExtendedTable, BypassArchiveLimits, MinSeverity, FixableOnly,
+		serverId, skipLogin, threads, detailedSummary, watches, repoPath, licenses, xrOutput, fail, ExtendedTable, BypassArchiveLimits, MinSeverity, FixableOnly, vuln,
```
Comment (eyalbe4) [inline utils/cliutils/commandsflags.go:1832]: What's the reason for this flag addition?

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: utils/cliutils/commandsflags.go (hunk 3 above)
Comment (EyalDelarea) [reply utils/cliutils/commandsflags.go:1832]: @eyalbe4 we noticed that this flag wasn't passed to JFrog CLI Security, so I added it here.

Comment (eyalbe4) [reply utils/cliutils/commandsflags.go:1832]: :+1:

**Pull request merged**